### PR TITLE
{vis}[foss/2021a] PyImageJ v1.3.1 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/468fb63b7a94c643b82660ba60f8cab6 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 31 out of 31 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux Ubuntu 20.04.2 LTS (Focal Fossa), x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/f216a470913bb4fb998fe9e49fc968ff for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/94902c45c4fe566ab1a68b94f760db8c for a full test report.

File: easybuild/easyconfigs/p/PyImageJ/PyImageJ-1.3.1-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('Java', '11', '', SYSTEM),
+    ('Maven', '3.6.3', '', SYSTEM),
+    ('matplotlib', '3.4.2'),
+    ('Pillow', '8.2.0'),
+    ('scikit-image', '0.18.3'),
+    ('SciPy-bundle', '2021.05'),
+    ('xarray', '0.19.0'),
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PyImageJ/PyImageJ-1.3.1-foss-2021a.eb:23]: @branfosj Never used this package, but does it make sense to add a dependency on `ImageJ`?

**Review by lexming — APPROVED** [review]
LGTM

File: easybuild/easyconfigs/p/PyImageJ/PyImageJ-1.3.1-foss-2021a.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/p/PyImageJ/PyImageJ-1.3.1-foss-2021a.eb:23]: A dependency on `ImageJ` would only get used if:
1. each user specifies to use that version of `ImageJ` every time they initialise
1. we made an edit to the code edits to default to that install

I am not sure the first option is sensible and that the second one is worthwhile.

Comment (lexming) [reply easybuild/easyconfigs/p/PyImageJ/PyImageJ-1.3.1-foss-2021a.eb:23]: I agree, a dependency on `ImageJ` will be too much of a constrain and it deviates significantly from the default workflow with `PyImageJ`. 

@verdurin what do you think?

Comment (verdurin) [reply easybuild/easyconfigs/p/PyImageJ/PyImageJ-1.3.1-foss-2021a.eb:23]: Sounds fine to me.

**Comment by lexming** [comment]
Going in, thanks @branfosj !

**Pull request merged**